### ISSUE TITLE
Animation Nodes won't install: This build doesn't work at the moment

### ISSUE THREAD

**Issue by zakski2** [JacquesLucke/animation_nodes#793]
At first I had problem of trying to install uncompiled version of the addon, like here - https://github.com/JacquesLucke/animation_nodes/issues/740. I deleted then all the files copied to the addons folder and downloaded Animation Nodes version for Windows 32 bit (which I use now) and tried to install addon. 
It tells that it's installed in the info log:
![info-window-install-log](https://user-images.githubusercontent.com/31315217/30757127-257bf97e-9fd6-11e7-8dac-53f2dd5786b2.jpg)
But once I try to activate it I get an error:
![error-main](https://user-images.githubusercontent.com/31315217/30757140-366cd0dc-9fd6-11e7-841a-ac379cb52c7b.jpg)
Messages in the console:
![console-error-log](https://user-images.githubusercontent.com/31315217/30757232-90f9a8e0-9fd6-11e7-9edf-d504442e1ef2.jpg)

I checked all possible reasons from the ones shown in the error and nothing helped so far. I had VC++ Redistributable 2015 installed, downloaded it once again and run as Repair to ensure it's installed.

Does this mean I can't use the addon on x32 bit system? (I saw a comment [here](https://github.com/JacquesLucke/animation_nodes/issues/792#issuecomment-331297615) which made me think kind of I can't)

I use Windows 7 x32 bit Home Extended Service Pack 1. The D disk partition is not system one, Program Files folder there is custom. Addons I install are located in the /Program Files/Blender-Addons/addons folder (this was done for sharing addons between all the installed versions of Blender).

**Comment by JacquesLucke** [comment]
So far I never compiled AN for a 32 bit system. It should be possible somehow but I don't know how without having access to a 32 bit system myself.

**Comment by zakski2** [comment]
Ah, I see. Now it's more clear, probably it should be mentioned somewhere for clarity (e.g. in the error message). I can't change OS now unfortunately.

**Comment by JacquesLucke** [comment]
yes that would be a good idea indeed! Will add that information ton the download page when I'm back home.
